### PR TITLE
:wq command

### PR DESCRIPTION
**Pull request by haydennyyy**
This command will write to the current file and then close the editor a la Vim

### CODE REVIEW

**Comment by haydennyyy** [comment]
Crap. Need to write tests.

**Comment by haydennyyy** [comment]
Sorry Anthony, might take me a while, I don't understand the testing framework lmao

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@
 /.pytest_cache
 /.tox
 /venv*
+/build
+/dist
```
Comment (asottile) [inline .gitignore]: also looks like your other commit snuck in here: https://github.com/asottile/babi/pull/10

Comment (haydennyyy) [reply .gitignore]: Ah crap. Erm, how do I remove that?

Comment (haydennyyy) [reply .gitignore]: I think this is resolved now.

Comment (asottile) [reply .gitignore]: you could rebase it out, `git rebase -i origin/master` and then delete the commits you don't want

**Comment by haydennyyy** [comment]
There we go, tests written.

File: tests/babi_test.py [hunk 2]
```diff
@@ -1019,3 +1019,15 @@ def test_cut_uncut_multiple_file_buffers(tmpdir):
...
+def test_write_and_quit(tmpdir):
+    f = tmpdir.join('f')
+
+    with run(str(f)) as h, and_exit(h):
+        h.press('hello world')
+        trigger_command_mode(h)
+        h.press_and_enter(':wq')
+        h.await_exit()
+
+    assert f.read() == 'hello world\n'
```
Comment (asottile) [inline tests/babi_test.py]: can you put this next to the other command mode tests?

Comment (haydennyyy) [reply tests/babi_test.py]: Done.

**Comment by haydennyyy** [comment]
I actually got a shipit this time. Nice.

**Pull request merged**